packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareSelectComponent.tsx

```diff
@@ -27,6 +28,15 @@ export interface VersionCompareSelectorProps {
 
   /** Named Versions to be displayed */
   namedVersions: CurrentNamedVersionAndNamedVersions | undefined;
+
+  /**
+  * Props for a href that will, on a click, navigate to the provided link or invoke the provided onClick method.
+  *
+  * Please note if href and both on click are provided; the component will not use on click but will use href instead.
+  *
+  * ManageNamedVersionLabel will default to `Manage named versions` if not provided.
```

Comment:
Nit: you can use TypeDoc to express the default value, e.g. `@default "Manage named versions"`